test/remote/gateways/remote_moneris_test.rb

```diff
@@ -141,6 +141,20 @@ def test_successful_purchase_with_network_tokenization_google_pay_source
     assert_false response.authorization.blank?
   end
 
+  def test_successful_purchase_with_network_tokenization_google_pay_source_without_order_id
+    @options[:order_id] = nil
+    @credit_card = network_tokenization_credit_card(
+      '4242424242424242',
+      payment_cryptogram: 'BwABB4JRdgAAAAAAiFF2AAAAAAA=',
+      verification_value: nil,
+      source: :google_pay
+    )
+    assert response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'Approved', response.message
+    assert_false response.authorization.blank?
+  end
```

Comment:
just a thing of format and background, for me I understood `without_order_id` that order_id is not present in options (that test already validated at line 131), the intent of this test is to check what happens if order_id has nil value, it is just what the description says about it.  It also could be excellent to add a test that validates the value of order_id remains exactly the same during the process due to that we have an order_id generation if the value is nil or not present.

Reply:
Sure, I made the changes and I added some extra tests. Thanks!